package excecao.execaoPersonalizada;

import excecao.Aluno;

public class TesteValidacoes {
	
	public static void main(String[] args) {
		
		Aluno aluno = new Aluno("Ana", -7);
		
		try {
			Validar.aluno(aluno);
		} catch (StringVaziaException e) {
			System.out.println(e.getMessage());
		}
		
		System.out.println("Fim :)");
	}
}
